cmd/oras/repository/tags.go

```diff
@@ -55,6 +55,12 @@ Example - Show tags of the target OCI layout folder 'layout-dir':
 
 Example - Show tags of the target OCI layout archive 'layout.tar':
   oras repo tags --oci-layout layout.tar
+
+Example - Show tags also associated with a particular tagged resource:
+  oras repo tags localhost:5000/hello:latest
```

Comment:
Shouldn't the `oras repo tags` prints the input tag?
```suggestion
Example - Show tags associated with a particular tagged resource:
  oras repo tags localhost:5000/hello:latest
```

Reply:
Done.